gabarit/template_nlp/nlp_project/package_name-scripts/utils/0_sweetviz_report.py

```diff
@@ -135,8 +135,8 @@ def get_paths_and_names(dataset_paths: List[str], dataset_names: List[str]) -> T
         dataset_paths (list<str>): List of datasets filenames (actually paths relative to {{package_name}}-data)
         dataset_names (list<str>): List of names to use for each dataset. If no name provided, backup on files names without extension.
     Raises:
-        AssertionError: If any provided dataset does not exist
-        AssertionError: If a provided list of name does not the correct number of datasets
+        FileNotFoundError: If any provided dataset does not exist
+        ValueError: If a provided list of name does not the correct number of datasets
```

Comment:
If a provided list of names does not matches/equals the correct number of datasets